src/index.js

```diff
@@ -171,11 +171,8 @@ class PayfitContentScript extends ContentScript {
           try {
             await this.autoLogin(credentials)
             this.log('info', 'autoLogin succesful')
-          } catch {
-            this.log(
-              'info',
-              'Something went wrong with autoLogin, letting user log in'
-            )
+          } catch (err) {
+            this.log('info', 'Something went wrong with autoLogin: ' + err)
```

Comment:
use err.message or else we won't see anything in the log message

Reply:
Sure, my bad !